docs/source/asr/models.rst

```diff
@@ -285,6 +285,31 @@ You may find the example config files of Conformer variant of such hybrid models
 with sub-word encoding at ``<NeMo_git_root>/examples/asr/conf/conformer/hybrid_transducer_ctc/conformer_hybrid_transducer_ctc_bpe.yaml``.
 
 
+.. _Conformer-HAT_model:
+
+Conformer-HAT (Hybrid Autoregressive Transducer)
+--------------------------------------
```

Comment:
The lines should have the same size as the title.

Reply:
Done